.github/ubiquibot-config.yml

```diff
@@ -28,5 +29,5 @@ priority-labels:
   - name: "Priority: 4 (Emergency)"
     weight: 5
 auto-pay-mode: true
-analytics-mode: true
+analytics-mode: false
```

Comment:
Why

Reply:
It is the default value of analytics mode. I am not sure why it has been changed. Let me revert it

Reply:
https://github.com/ubiquity/ubiquibot/pull/422/commits/55a0b32879330c281badf12dc4cb9ff9049d6151